### PR TITLE
Post: Metal3 at KubeCon EU 2024

### PR DESCRIPTION
**Pull request by lentzi90**
Add blog post about Metal3 presense at KubeCon EU 2024.

### CODE REVIEW

**Comment by tuminoid** [comment]
Besides technical matters, I'd like to see some sort of call to action added to that. Maybe link the good-first-issue search and call for people to check them out, since we got some left overs?

File: .cspell-config.json [hunk 1]
```diff
@@ -69,6 +69,8 @@
     "com",
     "ConfigMap",
     "contrib",
+    "contribfest",
+    "Contribfest",
```
[the remark this answers is not in the document]
Comment (lentzi90) [reply .cspell-config.json:73]: Good catch! It complained about all occurrences the first time and I just added both then. I will remove the capitalized one and see what happens

**Comment by lentzi90** [comment]
> Besides technical matters, I'd like to see some sort of call to action added to that. Maybe link the good-first-issue search and call for people to check them out, since we got some left overs?

Excellent idea!

File: .cspell-config.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (lentzi90) [reply .cspell-config.json]: Indeed no issue with only one of them

**Review by tuminoid — COMMENTED** [review]
/lgtm

Looks great to me now, thanks for the post!

**Comment by elfosardo** [comment]
/approve
I really like it, thanks!

**Pull request merged**